core/user/user.go

```diff
@@ -34,6 +34,11 @@ type User struct {
 	Disabled bool
 }
 
+// Filter is used to filter users.
+type Filter struct {
```

Comment:
Don't do this. We don't want any partially conceived DSL-based filtering. Add creator UUID as a parameter instead, and add a where clause in state if it is not empty.

Reply:
Got you, 

just push the change that drops Filter in favor of explicit creatorName.